locator/topology.cc

```diff
@@ -16,6 +16,24 @@
 #include "locator/production_snitch_base.hh"
 #include "utils/stall_free.hh"
 
+struct node_printer {
+    const locator::node* v;
+    node_printer(const locator::node* n) noexcept : v(n) {}
+};
+
+template <>
+struct fmt::formatter<node_printer> {
```

Comment:
Why not specialize for const node*?

Reply:
Because of

```
/usr/include/fmt/core.h:1751:3: error: static assertion failed due to requirement 'formattable_pointer': Formatting of non-void pointers is disallowed.
  static_assert(formattable_pointer,
```